src/plugins/GenerateJob/templates/backend_deepforge.py

```diff
@@ -301,6 +306,27 @@ def figure_to_state(self):
             state['axes'].append(axes_data)
         return state
 
+    def process_line_collection(self, collection):
+        line_collections = []
+        colors = collection.get_colors()
+        ls = collection.get_dashes()
+        lw = collection.get_linewidths()
+
+        for i, segment  in enumerate(collection.get_segments()):
+            line_collection_data = dict()
+            line_collection_data['points'] = segment.tolist()
+            label = collection.get_label()
+            if label is None:
+                line_collection_data['label'] = ''
+            else:
+                line_collection_data['label'] = label
+            line_collection_data['color'] = to_hex(colors[i%len(colors)])
+            line_collection_data['lineStyle'] = 'solid'
```

Comment:
Should this be hard coded?

Reply:
The method `collection.get_dahses` which returns tuple of `Nones`. Not sure whether it should be hardcoded. But, its not returning any property.

Reply:
alright. We can leave it as is for now and worry about it later.